--- a/src/driver/sqlx_sqlite.rs
+++ b/src/driver/sqlx_sqlite.rs
@@ -400,6 +400,12 @@
         macro_rules! try_get {
             ($v: expr, $t: ty) => {
                 |value| {
+                    // PATCH: Parse TEXT as NaiveDateTime for SQLite
+                    if let Ok(dt) = value.parse::<chrono::NaiveDateTime>() {
+                        return Ok(sea_orm::Value::DateTime(dt));
+                    } else if let Ok(dt) = chrono::NaiveDateTime::parse_from_str(value, "%Y-%m-%d %H:%M:%S%.f") {
+                        return Ok(sea_orm::Value::DateTime(dt));
+                    }
                     let v: Option<$t> = value.unwrap();
                     Ok(<$t as Decode<'_, sqlx::sqlite::Sqlite>>::decode(v)?)
                 }
